Header header

float32[] OF_meas
float32[] FOF_OF_SF
float32 FOF_yaw_rate_cmd
float32 FR_yaw_rate_cmd
float32[] FR_OF_SF

float64 vicon_position_x
float64 vicon_position_y
float64 vicon_position_z
float64 vicon_orientation_x
float64 vicon_orientation_y
float64 vicon_orientation_z
float64 vicon_orientation_w

float32 vicon_vel_x
float32 vicon_vel_y
float32 vicon_vel_z
float32 vicon_angular_z

float32 vicon_accel_x
float32 vicon_accel_y
float32 vicon_accel_z

float32 threshold
int16 dswitch



